.emacs.d/bin/omnisharp

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+lsb_release -ds | grep "Arch Linux" > /dev/null
+if [ $? -ne 0 ]; then
```

Comment:
Why not just merge L3 and L4 to get `if ! lsb_release -ds | grep "Arch Linux" > /dev/null;`?  I never trust `$?`, to be honest.